qndiag/tests/test_qndiag.py

```diff
@@ -39,3 +39,19 @@ def test_errors():
         x += x.swapaxes(1, 2)
         x[0] = np.array([[0, 1], [1, 0]])
         qndiag(x)
+
+@pytest.mark.parametrize('weights', [None])
```

Comment:
I would have just use a parametrize with ortho too and inside if ortho you simulate with an orthogonal matrix and you make sure the returned matrix is orthogonal.

it avoids some code dupe between tests